client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/AppiumCapabilitiesBuilder.java

```diff
@@ -30,8 +30,10 @@
     private static final String APPIUM_ANDROID_PLATFORM_TYPE = "ANDROID";
     private static final String APPIUM_IOS_PLATFORM_TYPE = "iOS";
     private static final String MOBILE_NODE_TYPE = "mobileNodeType";
-    private static final String APPIUM_LOCALE_TYPE = "locale";
-    private static final String APPIUM_LANGUAGE_TYPE = "language";
+    private static final String MOBILE_APP_MAIN_ACTIVITY = "appActivity";
+    private static final String MOBILE_APP_WAIT_ACTIVITY = "appWaitActivity";
+    private static final String MOBILE_APP_PACKAGE = "appPackage";
+    private static final String MOBILE_APP_WAIT_PACKAGE = "appWaitPackage";
```

Comment:
@mach6 appium 1.5.x with some of the apks, appium can not read the manifest and throws exception [1.5.2 - Android fails](https://github.com/appium/appium/issues/6499). the way to solve this issue is by defining package and main activity of the application. i was not able to find the official document to reference to, but if you run appium ui and set these 2 parameters, it will print in console:

    --app-pkg => --default-capabilities '{"appPackage":"com.paypal.selion.pageobjectsdemoapp"}'
    --app-activity => --default-capabilities '{"appActivity":".PageObjectsHomeActivity"}'

2 other parameters are, wait for the same package and activity to start :)